narwhals/series.py

```diff
@@ -1835,11 +1847,14 @@ def get_categories(self) -> Series:
                "mango"
             ]
         """
-        return self._series.__class__(self._series._series.cat.get_categories())
+        return self._series.__class__(
+            self._series._series.cat.get_categories(), api_version=self._api_version
+        )
```

Comment:
Is it worth using 
```suggestion
        return self._from_series(self._series._series.cat.get_categories())
```
here and in many other `self._series.__class__` calls? It would avoid passing `self._api_version` all the time

Reply:
yup, good one thanks!